Dockerfile

```diff
@@ -16,12 +16,15 @@ RUN npm run build
 
 FROM node:16-alpine3.15 
 RUN apk add curl
-WORKDIR /root
+WORKDIR /app
 ADD package*.json ./
 RUN npm install --production
 COPY --from=solidity-builder /home/node/contracts contracts/source
 COPY --from=solidity-builder /home/node/artifacts/contracts/ERC1155MixedFungible.sol contracts
 COPY --from=builder /root/dist dist
-COPY --from=builder /root/.env /root/.env
+COPY --from=builder /root/.env /app/.env
+RUN chgrp -R 0 /app/ \
+    && chmod -R g+rwX /app/
+USER 1001
```

Comment:
I think this one might have the same problem as the other PR that you fixed

Reply:
It should not because in this one we're assuming the `USER` till after we've done everything else.

I may refactor one or the other Dockerfiles to be more consistent with _when_ we do `npm i`'s and with what users once I've tested out these images at runtime. Because right now these similar runtimes go about their dockerfiles / bundling a bit different for reasons I don't yet appreciate.